bin/bootstrap-cloud

```diff
@@ -192,9 +213,120 @@ if [[ ${__cloud} == aws ]]; then
 	-e "s#some-context#aws-${__name}#" \
         env.auto.tfvars.example > env.auto.tfvars
   fi
+  ;;
+  gcp)
+    check_gcp_prerequisites
+    
+    # Config file has to exist at this point
+    if [[ ! -e ${__root}/config.yaml ]]; then
+      echo "config.yaml not found, exiting..."; exit 1
+    fi
+    # Get the project ID
+    __project_id=$(grep ^terraform_project_id: ${__root}/config.yaml | awk '{print $2}')
+    if [[ -z ${__project_id} ]]; then
+      __project_id=$(gcloud config get-value project)
+      echo "terraform_project_id: ${__project_id}" >> ${__root}/config.yaml
```

Comment:
Don't write to the user's configuration file. Consider it immutable (by us). If something *must* be set in the configuration, error and force the user to do it.